### ISSUE TITLE
EXCEPTION occurred: Batch normalization training on CPU is not yet implemented.

### ISSUE THREAD

**Issue by vermorel** [Microsoft/CNTK#2278]
I am trying to run a `BatchNormalizationLayer` in BrainScript but, when attempting to execute on CPU, I get the following error: 

    EXCEPTION occurred: Batch normalization training on CPU is not yet implemented.

The batch normalization is very important, and it's a real friction not being able to even test training a model without a GPU.

**Comment by spandantiwari** [comment]
We not have batch normalization on CPU. There is a work item open for enabling batch normalization on CPU for inference only. I will mark this as a feature request, but we do not have an ETA for this.

**Comment by rringham** [comment]
+1 as a feature request

**Comment by KeDengMS** [comment]
[Fixed](https://github.com/Microsoft/CNTK/commit/1d3d1e8a2770b65a809f6bf7e6cde49047ffc0c6)
